src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,42 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int STACK_FULL_VALUE = 11;
 
-	public int total = -1;
+	private static final int STACK_EMPTY_VALUE = -1;
 
-	public void countIn(int in) {
+	private static final int STACK_MAX_CAPACITY = 12;
+
+	private final int[] numbers = new int[STACK_MAX_CAPACITY];
+
+	private int total = STACK_EMPTY_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)